### PR TITLE
Fix broken network spec

### PR DESCRIPTION
**Pull request by borod108**
The ovn spec was broken due to openstack provider changes.
Made a new recording.

Note currently the ovn did not respond to some of the requests that had
limit=1000 as a parameter, this is a bug on the ovn side and will be
handled separately, the current recording was actually modified as if
the bug did not exist. This does not reflect on the changes introduced
in this commit but I thought it is important to note.

### CODE REVIEW

File: spec/models/manageiq/providers/redhat/infra_manager/refresh/refresher/response_yamls/external_network_providers.yml [hunk 1]
```diff
@@ -6,11 +6,11 @@ internal:
   description: 
   id: 6e452166-e772-4355-b050-24705a32c15c
   name: ovn
-  authentication_url: http://localhost:35357/v2.0/
+  authentication_url: http://bodnopoz-engine.eng.lab.tlv.redhat.com:35357/v2.0/
```
Comment (agrare) [inline spec/models/manageiq/providers/redhat/infra_manager/refresh/refresher/response_yamls/external_network_providers.yml]: Can you change these back to localhost?  Best to not have internal DNS names in public spec helpers.

Comment (borod108) [reply spec/models/manageiq/providers/redhat/infra_manager/refresh/refresher/response_yamls/external_network_providers.yml]: done

**Comment by agrare** [comment]
@borod108 it looks like the tests are still failing with an openstack handle error, is that what you were trying to fix?

**Comment by borod108** [comment]
@masayag please review

**Comment by simaishi** [comment]
@borod108 @masayag Travis is red for Hammer branch, and I'm wondering if we need this PR in hammer branch too?

https://travis-ci.org/ManageIQ/manageiq-providers-ovirt/builds/457466378

**Comment by simaishi** [comment]
@agrare ^ can you help?

**Comment by agrare** [comment]
@simaishi think you're right looks like the same failure

**Comment by simaishi** [comment]
Thanks Adam.

Hammer backport details:

```
$ git log -1
commit adc6bfab1686645db2078955c373e18a7233ed53
Author: Moti Asayag <masayag@redhat.com>
Date:   Thu Nov 15 12:25:58 2018 +0200

    Merge pull request #309 from borod108/specs/fix_broken_ovn
    
    Fix broken network spec
    
    (cherry picked from commit 5e28c1d7ab71f451d8aeb53e28191513a213dcd3)
```

**Comment by agrare** [comment]
Aaaaand we've progressed to a different error

**Comment by agrare** [comment]
@borod108 can you help out? I don't recognize that error

**Comment by simaishi** [comment]
```
     VCR::Errors::UnusedHTTPInteractionError:
       There are unused HTTP interactions left in the cassette:
```

https://travis-ci.org/ManageIQ/manageiq-providers-ovirt/jobs/457528101

**Pull request closed without merging**